.github/CODEOWNERS

```diff
@@ -1899,6 +1899,7 @@
 /types/fluent__sequence/                                @blushingpenguin
 /types/flush-write-stream/                              @djcsdy
 /types/flushable/                                       @pash90
+/types/flutterwave-node/                                @besemuna
```

Comment:
This is a generated file and will be updated automatically.  Please revert this change.

Reply:
I would do that now